src/main/java/ua/danit/final_project/entities/Employee.java

```diff
@@ -54,6 +55,11 @@
   @Column(name = "e_info")
   private String info;
 
+  /*  @ToString.Exclude
+  @EqualsAndHashCode.Exclude
+  @OneToMany(mappedBy = "employee", cascade = CascadeType.ALL)
+  private List<ShiftComment> comments;*/
```

Comment:
![Code Smell](https://sonarsource.github.io/sonar-github/code_smell.png 'Code Smell') Code Smell: This block of commented-out lines of code should be removed. ([squid:CommentedOutCodeLine](https://sonarcloud.io/organizations/final-project/rules#rule_key=squid%3ACommentedOutCodeLine))

[See it in SonarCloud](https://sonarcloud.io/project/issues?id=final-project&pullRequest=147&issues=AWaL1GS0Vfn4NtZOtxmr&open=AWaL1GS0Vfn4NtZOtxmr)

[ISSUE_KEY]: AWaL1GS0Vfn4NtZOtxmr